src/views/forms/AccountManagementFormsNew/WithdrawForm/WithdrawForm.tsx

```diff
@@ -121,7 +119,8 @@ export const WithdrawForm = () => {
     setFromTokenDenom(usdcDenom);
     // Cosmos chains connect to the keplr wallet, which has a unique address per chain id
     const calculatedCosmosAddress = toChainId && cosmosChainAddresses[toChainId];
-    setToAddress(calculatedCosmosAddress ?? sourceAccount.address);
+    // If there is an exchange, we cannot know what to populate the address as
+    setToAddress(exchangeName ? undefined : calculatedCosmosAddress ?? sourceAccount.address);
```

Comment:
i think it only makes sense to do this if the selected chain matches the chain of the sourceAccount (eg dont fill in solana address for evm withdrawals)

Reply:
i'm actually going to remove this completely for now - technically the figma mocks want a button that does this, so we don't need to be doing this by default. i forgot this was mostly done for convenience sake.

**ALTHOUGH**... some other exchanges make the assumption you are withdrawing to your source connected and force you to do so. i think there's some merit to this (although that's probably a convo for later) since we **have** had at least one instance where a user withdrew to the incorrect address and lost their funds 💀